website/blog/2026/02/02-18-simplify-multi-cluster-configuration-with-static-manifest-propagation.md

```
@@ -0,0 +1,100 @@
+- avatar: https://avatars.githubusercontent.com/rfranzke
+  email: rafael.franzke@sap.com
+  login: rfranzke
+  name: Rafael Franzke
+aliases: ["/blog/2026/02/18/simplify-multi-cluster-configuration-with-static-manifest-propagation"]
+---
+
+Managing configurations consistently across a fleet of Kubernetes clusters can be a complex task. Operators often need a straightforward way to deploy baseline resources—such as security policies, resource quotas, or RBAC rules—to all or a subset of their clusters without the overhead of building and maintaining a full-blown extension.
+
+Gardener now introduces a new feature that directly addresses this need: **Static Manifest Propagation from Seeds to Shoots**. This enhancement provides a declarative, centralized mechanism for distributing Kubernetes manifests to Shoot clusters.
+
+### How It Works
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)